internal/js/summary.js

```diff
@@ -196,6 +199,7 @@ class ReportBuilder {
 		this.renderContext = renderContext;
 		this.options = options;
 		this.sections = [];
+		this.globalMaxNameWidth = null;
```

Comment:
Should this be `0`?

Reply:
It probably should, indeed 👍🏻 Fixing it 🚀